plaso/cli/helpers/event_filters.py

```diff
@@ -35,12 +34,14 @@ def AddArguments(cls, argument_group):
           argparse group.
     """
     argument_group.add_argument(
-        '--slice', metavar='DATE', dest='slice', type=str, default='',
+        '--slice', metavar='DATE_TIME', dest='slice', type=str, default='',
         action='store', help=(
-            'Create a time slice around a certain date. This parameter, if '
-            'defined will display all events that happened X minutes before '
-            'and after the defined date. X is controlled by the parameter '
-            '--slice_size but defaults to 5 minutes.'))
+            'Date and time to create a time slice around. This parameter, if '
+            'defined, will display all events that happened X minutes before '
+            'and after the defined date, where X is controlled by the '
+            '--slice_size option, which is 5 minutes by default. The date and '
+            'time must be specified in ISO 8601 format including time zone '
```

Comment:
Please add an example of what an ISO 8601 formatted date and time looks like.

Reply:
ack